src/integ_test_resources/ios/sdk/integration/cdk/cdk_integration_tests_ios/iot_stack.py

```diff
@@ -214,20 +293,20 @@ def setup_custom_authorizer_function(self) -> str:
         """
         authorizer_function = aws_lambda.Function(
             self,
-            "iot_custom_authorizer_function",
+            "iot_custom_authorizer_function_{}".format(unique_id),
```

Comment:
I've been favoring the `f"{}"` construction of formatted strings since it's a bit terser, and puts the interpolated values inline with where they're used so we don't have to scan the `format` arguments and count the brace pairs to figure out what goes where: `f"iot_custom_authorizer_function_{unique_id}"`